app/controllers/api/v1/application_controller.rb

```diff
@@ -1,2 +1,78 @@
 class Api::V1::ApplicationController < ActionController::API
+
```

Comment:
We could try to move these lines to concerns:
http://richonrails.com/articles/rails-4-code-concerns-in-active-record-models